kernel/module/main.c

```diff
@@ -2739,7 +2739,7 @@ static int unknown_module_param_cb(char *param, char *val, const char *modname,
  * zero, and we rely on this for optional sections.
  */
 static int load_module(struct load_info *info, const char __user *uargs,
-		       int flags, bool no_args)
```

Comment:
As load_moudle is a common API in kernel/module/main.c, if update the parameter format, will it impact other load module process?

Reply:
no, 
it's only extending for dGPU driver，
legacy working no changes.